bin/katello-certs-check

```diff
@@ -124,6 +124,30 @@ function check-non-ascii () {
     fi
 }
 
+
+function check-server-cert-encoding () {
+    printf "Checking server certificate's encoding: "
+    openssl x509 -inform pem -in $CERT_FILE -noout -text &> /dev/null
+    if [ $? == "0" ]; then
+        success
+    else
+        openssl x509 -inform der -in $CERT_FILE -noout -text &> /dev/null
+        if [ $? == "0" ]; then
+            printf "[FAIL]\n"
```

Comment:
I have amended the error handling to match that of the other functions. Specifically, the `error` function is now called to display the first line of the error message, then the remainder of the error message is output.

I believe I have addressed and resolved the concern initially raised about the PR.